src/libponyc/codegen/genexe.c

```diff
@@ -269,9 +268,9 @@ static bool link_exe(compile_t* c, ast_t* program,
 #ifdef PONY_USE_LTO
     "-flto -fuse-linker-plugin "
 #endif
-    "%s %s %s %s -lpthread %s "
-    "-lm",
-    linker, file_exe, arch, mcx16_arg, fuseld, file_o, lib_args, ponyrt, ldl
+    "-fuse-ld=gold %s %s %s -lpthread %s -lm "
```

Comment:
Should `-fuse-ld=gold` always be set? Is that option available on FreeBSD? Is it consistently available on Linux? The options for `PONY_USE_LTO` had to be changed several times to handle various combinations of things.